### PR TITLE
*:improve code readability and unified aliases

### PR DESCRIPTION
**Pull request by yangwenmai**
For code readability and unified aliases.
I replace `"github.com/pingcap/tipb/go-tipb"` to `tipb "github.com/pingcap/tipb/go-tipb"`.

### CODE REVIEW

**Comment by zz-jason** [comment]
@yangwenmai There are some irrelevant changes, I think we'd better remove them and only replace `"github.com/pingcap/tipb/go-tipb"` with `tipb "github.com/pingcap/tipb/go-tipb"` in this PR.

**Comment by shenli** [comment]
@yangwenmai Thanks! Please fix the CI.

**Comment by zz-jason** [comment]
@yangwenmai functions `Next()` was removed in this PR: https://github.com/pingcap/tidb/pull/5993

**Comment by zz-jason** [comment]
@yangwenmai plz merge master and resolve conflicts.

**Comment by zz-jason** [comment]
@yangwenmai plz run `make check` to fix ci.

**Comment by yangwenmai** [comment]
I don't know what is wrong.
I run `make check`, no error.

File: distsql/distsql_test.go [hunk 1]
```diff
@@ -14,7 +14,7 @@
 package distsql
 
 import (
-	. "github.com/pingcap/check"
+  . "github.com/pingcap/check"
```
Comment (coocood) [inline distsql/distsql_test.go]: This is not aligned.

Comment (coocood) [reply distsql/distsql_test.go]: `go fmt ./...` under the root path of the repo can fix it.

**Pull request merged**